CHANGELOG.md

```diff
@@ -95,6 +95,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - added extra error page when current domain cannot be resolved with explicit hint about `overwrite_domain_url` parameter (for TEST environment)
     - using standard redis separators (`:`) in prefixes
     - build version is used in redis prefix of the twig cache and contains the environment (avoids the usage of a wrong cache)
+    - translations are extracted from overwritten templates now
```

Comment:
are they? won't we also extract them via phing target?

Reply:
3c469ca41